package.json

```diff
@@ -145,7 +145,8 @@
             },
             {
                 "command": "language-julia.executeFile",
-                "title": "Julia: Execute File"
+                "title": "Julia: Execute File",
+                "icon": "$(play)"
```

Comment:
Unrelated fix to make sure we display an icon above the editor to run a file instead of the command's title.